content/en/docs/reference/issues-security/official-cve-feed.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Official CVE Feed
+weight: 25
+outputs:
+  - json
+  - html 
+layout: official-cve-feed
+---
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+This is a community maintained list of official CVEs announced by
+Kubernetes Security Response Committee. 
+
+A programmatically accessible `JSON` Feed can be found [here](./index.json).
+You can access it by executing the following command:
+  ```shell
+  curl -v https://kubernetes.io/docs/reference/issues-security/official-cve-feed/index.json
+  ```
+
+{{< official-cve-feed >}}
```

Comment:
After alpha, it'd be nice to sort the table that this renders.

Reply:
Yes agreed. I have added it in future scope here https://github.com/kubernetes/sig-security/issues/1